upgrade/breaking-changes/4-0-0 .md

```diff
@@ -0,0 +1,308 @@
+---
+title: 4.0.0
+description: Handle the changes in the43.0.0 release of the Telerik UI for Blazor components.
+page_title: Breaking Changes in 4.0.0
+slug: changes-in-4-0-0
+position: 1
+---
+
+# Breaking Changes in 4.0.0
+
+## Common Changes
+
+### Supported .NET versions
+
+Microsoft drops the support of `.NET Core 3.1.x` on December 13, 2022. From the 4.0.0 release, Telerik UI for Blazor will support `.NET 6` and newer versions.
+
+### Stretch enum value
```

Comment:
It may be easier for customers to recognize the old API member names, rather than the new ones. This will make the section look more relevant to the reader. What do you think?

```suggestion
### Stretched enum Value
```